### ISSUE TITLE
Using arrows in virtualized and collapsed DropDownList/ComboBox to navigate through items cannot load next portion of items when reached

### ISSUE THREAD

**Issue by nenchef** [telerik/kendo-ui-core#5467]
### Bug report

Using arrows in virtualized DropDownList/ComboBox to navigate through items cannot load the next portion of items when reached

### Reproduction of the problem
Use the online demos: https://demos.telerik.com/kendo-ui/dropdownlist/virtualization
1. Focus the DropDown
2. Collapse the expanded dropdown
3. Use the Down arrow to move through the items collection

### Current behavior
Then the end of the currently loaded portion of items is hit - the navigation starts from the first item again.

### Expected/desired behavior
The next portion of items should be loaded, as it works in a scenario with expanded dropdown

### Environment

* **Kendo UI version:** 2019.3.1023
* **Browser:** [all]
